src/app/3d/qgs3dmapcanvas.cpp

```diff
@@ -119,6 +119,8 @@ void Qgs3DMapCanvas::setMap( Qgs3DMapSettings *map )
   connect( mScene, &Qgs3DMapScene::fpsCountChanged, this, &Qgs3DMapCanvas::fpsCountChanged );
   connect( mScene, &Qgs3DMapScene::fpsCounterEnabledChanged, this, &Qgs3DMapCanvas::fpsCounterEnabledChanged );
   connect( mScene, &Qgs3DMapScene::viewed2DExtentFrom3DChanged, this, &Qgs3DMapCanvas::viewed2DExtentFrom3DChanged );
+  // update the navigation widget when the near far plane has been updated by the map scene
+  connect( mScene, &Qgs3DMapScene::nearFarPlaneChanged, mNavigationWidget, &Qgs3DNavigationWidget::updateFromCamera );
```

Comment:
I don't think we need a new signal here. mScene->cameraController()->camera() returns Qt3DRender::QCamera, which has nearPlaneChanged() and farPlaneChanged() signals - let's use those!

Reply:
Done.